src/main/java/com/google/api/codegen/nodejs/NodeJSGapicContext.java

```diff
@@ -68,6 +68,13 @@ public String filePath(ProtoFile file) {
   }
 
   /**
+   * The package name of the grpc module for the API.
+   */
+  public String grpcClientName(Interface service) {
+    return "grpc-" + service.getParent().getFullName().replace('.', '-');
```

Comment:
optional: I prefer the syntax `service.getFile()` rather than `service.getParent()` (same for other proto elements) wherever possible because it seems more readable/descriptive.